app/web/features/auth/password/CompleteResetPassword.stories.tsx

```diff
@@ -1,31 +1,22 @@
-import { Meta, Story } from "@storybook/react";
 import { Empty } from "google-protobuf/google/protobuf/empty_pb";
-import React from "react";
+import { Meta, Story } from "@storybook/react";
 import { mockedService } from "stories/serviceMocks";
+import React from "react";
 
-import CompleteResetPassword from "./CompleteResetPassword";
+import SetNewPassword from "./CompleteResetPassword";
 
 export default {
-  component: CompleteResetPassword,
-  title: "Me/Auth/CompleteResetPasswordPage",
+  component: SetNewPassword,
+  title: "Me/Auth/SetNewPasswordPage",
```

Comment:
other components have the `Page` suffix at the end... I guess that `CompleteResetPasswordPage` it's ok? (since the component name it's `CompleteResetPassword`